### PR TITLE
API Downgrade webpack and babel using @silverstripe/webpack-config for module consistency

### PR DESCRIPTION
**Pull request by ScopeyNZ**

### CODE REVIEW

**Review by robbieaverill — COMMENTED** [review]
LGTM, if you could fix the commit message typo that'd be sweet though

**Comment by ScopeyNZ** [comment]
Done. Also the amended commit removes `import compose from 'redux'` and instead inline's the compose function.

**Comment by ScopeyNZ** [comment]
Also just quickly rebased

File: src/dependency-injection/buildBaseContainer.js [hunk 1]
```diff
@@ -1,5 +1,16 @@
 import MiddlewareRegistry, { GLOBAL_CONTEXT } from './MiddlewareRegistry';
-import { compose } from 'redux';
+
+function compose(...funcs) {
```
Comment (robbieaverill) [inline src/dependency-injection/buildBaseContainer.js:4]: Do you want to add a comment saying you've borrowed this from redux, just for clarity?

**Review by Cheddam — APPROVED** [review]
Want to publish `0.1.2`, and include the version change in this PR? Otherwise good to go!

**Comment by ScopeyNZ** [comment]
Thanks for the reminder. I've pushed a commit that bumps the version number.

**Pull request merged**